README.md

```diff
@@ -1,132 +1,164 @@
 # SwiftFusion
 
-Differentiable Swift based sensor fusion library. 
+A differentiable sensor fusion library written in Swift.
 
-Think factor graphs a la [GTSAM](https://gtsam.org/) coupled with deep learning, [Jax](https://github.com/google/jax)/[TensorFlow](https://www.tensorflow.org/) style. Based on [Swift for TensorFlow](https://www.tensorflow.org/swift).
+Think factor graphs (à la [GTSAM](https://gtsam.org)) coupled with deep learning (via [Swift for TensorFlow](https://www.tensorflow.org/swift)).
 
-Still very early, but feel free to explore! Subject to *massive* change :-)
+This project is in an early phase, but feel free to explore! Subject to *massive* changes. :-)
 
-## Getting Started
+## Getting started
 
-### Swift on MacOS
+### Installation
 
-Using XCode is the easiest way to develop on Mac. Unfortunately, the current version of Xcode 12 has a bug, and you need to work with the XCode beta, available from Apple [here](https://developer.apple.com/download/).
+#### macOS
 
-Follow the instructions to [install Swift for TensorFlow on MacOS](https://github.com/tensorflow/swift/blob/master/Installation.md#macos). Installing the latest development snapshot is recommended.
+Using Xcode is the easiest way to develop on macOS. Unfortunately, the current version of Xcode 12 has a bug, and you need to work with the Xcode beta, available from Apple [here](https://developer.apple.com/download/).
```

Comment:
Just wonder if this is still true, cause I am using 12.1 right now and it works fine?

Reply:
Thanks for pointing this out! I updated the blurb about Xcode version to be more accurate:

> Download a [Swift for TensorFlow macOS toolchain](https://github.com/tensorflow/swift/blob/main/Installation.md#development-snapshots) and follow the [macOS installation instructions](https://github.com/tensorflow/swift/blob/master/Installation.md#macos). Installing the latest development snapshot is recommended. [Check here](https://github.com/apple/swift/blob/main/docs/HowToGuides/GettingStarted.md#installing-dependencies) for the "required" version of Xcode for using nightly toolchains. Sometimes toolchains may also work with older Xcode minor versions than the one listed.